docs/spec_draft.md

```diff
@@ -705,6 +706,57 @@ lexicographic comparison on the (real, imaginary) pairs.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.multiply
+
+### Semantics
+
+Performs element-wise product of two tensors `lhs` and `rhs` and produces a
+`result` tensor. For integer element types, if the element-wise product has an
+unsigned/signed overflow/underflow, the result is implementation-defined and one
+of the following:
+
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
+  for unsigned overflow/underflow. For signed integer overflow/underflow, wraps
+  the result around the representable range $[-2^{n-1},\ \ 2^{n-1} - 1]$.
+  * saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow (or signed
+  underflow) and saturation to $2^n - 1$ (or $0$) for unsigned overflow (or
+  unsigned underflow).
+
+For floating-point element types, it implements the `multiplication` operation from
+the IEEE-754 specification.
+
+For complex element types, it computes a complex multiplication, with corner cases TBD.
+
+### Operands
+
+| Name  | Type                                                        |
+|-------|-------------------------------------------------------------|
+| `lhs` | tensor of integer, floating-point, or complex element types |
+| `rhs` | tensor of integer, floating-point, or complex element types |
+
+### Results
+
+| Name     | Type                                                        |
+|----------|-------------------------------------------------------------|
+| `result` | tensor of integer, floating-point, or complex element types |
+
+### Constraints
+
+  * (C1) `lhs`, `rhs` and `result` have the same type.
+
+### Examples
+
+```mlir
+// %lhs: [[1, 2], [3, 4]]
```

Comment:
Since the type is `f32`:
`// %lhs: [[1.0, 2.0], [3.0, 4.0]]`
Here and below.

Reply:
changing to i32